onboarding/time-management.md

```diff
@@ -67,4 +67,18 @@ The costs of multitasking include:
 - **Quality:** Trying to do too much at once lowers the quality of your work.
 - **Stress:** Constant task-switching can create added stress.
 
-##  Supplemental Materials
\ No newline at end of file
+##  Supplemental Materials
+There are a lot of amazing time management tips come from 
+behavioral ADHD advice — check out these resources!
+
+### Helpful Readings to Consider 
+- [Tips for Managing ADHD](https://www.dropbox.com/scl/fi/8rs1jl531nswipemldrt2/tips-for-managing-adhd.pdf?rlkey=xnbuwseumrhggtw6m8q8mw0ys&st=zj6quy2j&dl=0)
+- [Mastering Your Adult ADHD: A Cognitive-Behavioral Treatment Program ](https://www.dropbox.com/scl/fi/n69qpj0j34gs64utgpt86/Mastering-Your-Adult-ADHD_-A-Cognitive-Behavioral-Treatment-Program-Client-Workbook-PDFDrive.pdf?rlkey=srofcoszvfqq16cbzhvfxsamk&st=l2ojer7n&dl=0)
+
+- [ Managing ADHD ](https://www.dropbox.com/scl/fi/4kqo4dsw1ikfzuaczl1qt/managing-adhd.pdf?rlkey=an7jmdhi0p4aubgbo1q4pwqmi&st=fuse28fe&dl=0)
+
+- [Identifying Feelings with the Hoffman list](https://www.hoffmaninstitute.org/wp-content/uploads/Practices-FeelingsSensations.pdf)
```

Comment:
```suggestion
- [Identifying Feelings with the Hoffman list](https://www.hoffmaninstitute.org/wp-content/uploads/Practices-FeelingsSensations.pdf) (PDF)
```

Reply:
Hi @mai-repo can you leave a comment or even a commit hash clarifying where this has been addressed? Please do this before resolving each change request